[CI] MixedClusterClientYamlTestSuiteIT test {p0=mapping/70_columnar_prefix_dynamic_properties/columnar: no prefix_properties when dynamic is only set at root level} failing

**Build Scans:**
- [elasticsearch-intake #43179 / 9.5.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/g2vhruqcs6xi6)
- [elasticsearch-pull-request #161351 / 8.19.19_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/a3kn7uzhkipzo)
- [elasticsearch-pull-request #161351 / 9.3.8_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/nnb2pi4le75cu)
- [elasticsearch-pull-request #161351 / 9.4.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/ykwc5dz55fsqi)
- [elasticsearch-pull-request #161351 / 8.19.19_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/mqjt5cjt4jlxe)
- [elasticsearch-pull-request #161310 / 9.4.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/nuevhx7eubcbi)
- [elasticsearch-pull-request #161310 / 9.3.8_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/lj5xyszttksaq)
- [elasticsearch-pull-request #161259 / 9.4.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/pwag4y5m2o626)
- [elasticsearch-pull-request #161259 / 9.3.8_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/3mbwqp27qspns)
- [elasticsearch-pull-request #161259 / 9.3.8_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/4w4tnpejkq66s)

**Reproduction Line:**
```
./gradlew ":qa:mixed-cluster:v9.5.0#mixedClusterTest" -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=mapping/70_columnar_prefix_dynamic_properties/columnar: no prefix_properties when dynamic is only set at root level}" -Dtests.seed=E936DA88B5482032 -Dtests.bwc=true -Dtests.locale=ar-PS -Dtests.timezone=Australia/Hobart -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Dmapping%2F70_columnar_prefix_dynamic_properties%2Fcolumnar%3A%20no%20prefix_properties%20when%20dynamic%20is%20only%20set%20at%20root%20level%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: java.lang.AssertionError: field [test_columnar_root_only_dynamic.mappings.properties.version] doesn't have a true value
Expected: not null
     but: was null
```

**Issue Reasons:**
- [main] 1 consecutive failures in test test {p0=mapping/70_columnar_prefix_dynamic_properties/columnar: no prefix_properties when dynamic is only set at root level}
- [main] 1 consecutive failures in step 9.5.0_bwc-snapshots
- [main] 3 consecutive failures in step 8.19.19_part_6_bwc-snapshots
- [main] 4 consecutive failures in step 9.3.8_part_6_bwc-snapshots
- [main] 3 consecutive failures in step 9.4.4_part_6_bwc-snapshots
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 11 failures in test test {p0=mapping/70_columnar_prefix_dynamic_properties/columnar: no prefix_properties when dynamic is only set at root level} (2.7% fail rate in 410 executions)
- [main] 3 failures in step 8.19.19_part_6_bwc-snapshots (100.0% fail rate in 3 executions)
- [main] 4 failures in step 9.3.8_part_6_bwc-snapshots (100.0% fail rate in 4 executions)
- [main] 3 failures in step 9.4.4_part_6_bwc-snapshots (100.0% fail rate in 3 executions)
- [main] 3 failures in pipeline elasticsearch-pull-request (1.1% fail rate in 270 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -319,9 +319,6 @@ tests:
 - class: org.elasticsearch.xpack.transform.integration.TransformCrossProjectIT
   method: testUpdateTransformWithProjectRouting
   issue: https://github.com/elastic/elasticsearch/issues/147534
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: "test {p0=mapping/70_columnar_prefix_dynamic_properties/columnar: no prefix_properties when dynamic is only set at root level}"
-  issue: https://github.com/elastic/elasticsearch/issues/153390
 - class: org.elasticsearch.upgrades.DataStreamsUpgradeIT
   method: testDataStreamValidationDoesNotBreakUpgrade
   issue: https://github.com/elastic/elasticsearch/issues/153413
@@ -397,9 +394,6 @@ tests:
 - class: org.elasticsearch.xpack.ccr.CcrSeqNoPruningIT
   method: testSeqNoPartiallyRetainedByCcrLease
   issue: https://github.com/elastic/elasticsearch/issues/152498
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: "test {p0=mapping/70_columnar_prefix_dynamic_properties/logsdb_columnar: no prefix_properties when dynamic is only set at root level}"
-  issue: https://github.com/elastic/elasticsearch/issues/154062
 - class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecSpatialShapesIT
   method: test {csv-spec:spatial_shapes.convertFromStringParseError}
   issue: https://github.com/elastic/elasticsearch/issues/154157
@@ -415,12 +409,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.optimizer.PhysicalPlanOptimizerTests
   method: testMaxExpressionDepth_nestedAbs_maxAllowed {default}
   issue: https://github.com/elastic/elasticsearch/issues/154380
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: "test {p0=mapping/70_columnar_prefix_dynamic_properties/logsdb_columnar: multiple sub-objects with all three supported dynamic values}"
-  issue: https://github.com/elastic/elasticsearch/issues/154464
-- class: org.elasticsearch.smoketest.SmokeTestMultiNodeClientYamlTestSuiteIT
-  method: "test {yaml=mapping/70_columnar_prefix_dynamic_properties/logsdb_columnar: no prefix_properties when dynamic is only set at root level}"
-  issue: https://github.com/elastic/elasticsearch/issues/154537
 - class: org.elasticsearch.xpack.remotecluster.RemoteClusterSecurityCcrMigrationIT
   method: testRcs1SetupAgain
   issue: https://github.com/elastic/elasticsearch/issues/154539
@@ -547,9 +535,6 @@ tests:
 - class: org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT
   method: test {yaml=downsample/100_downsample_histograms/Downsample histogram with last value}
   issue: https://github.com/elastic/elasticsearch/issues/155453
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: "test {p0=mapping/70_columnar_prefix_dynamic_properties/columnar: multiple sub-objects with all three supported dynamic values}"
-  issue: https://github.com/elastic/elasticsearch/issues/155501
 - class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlSpecKnnFunctionIT
   method: test {csv-spec:knn-function.knnAfterChainedRename}
   issue: https://github.com/elastic/elasticsearch/issues/155518

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/mapping/70_columnar_prefix_dynamic_properties.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/mapping/70_columnar_prefix_dynamic_properties.yml
@@ -38,6 +38,10 @@ setup:
           version: "9.5.0"
 
   - do:
+      cluster.health:
+        wait_for_events: languid
+
+  - do:
       indices.get_mapping:
         index: test_columnar_root_only_dynamic
   - is_true: test_columnar_root_only_dynamic.mappings.properties.version
@@ -148,6 +152,10 @@ setup:
             term:
               resource.service: "my-service"
   - match: { hits.total.value: 1 }
+
+  - do:
+      cluster.health:
+        wait_for_events: languid
 
   - do:
       indices.get_mapping:
@@ -369,6 +377,10 @@ setup:
           version: "9.5.0"
 
   - do:
+      cluster.health:
+        wait_for_events: languid
+
+  - do:
       indices.get_mapping:
         index: test_logsdb_columnar_root_only_dynamic
   - is_true: test_logsdb_columnar_root_only_dynamic.mappings.properties.@timestamp
@@ -416,6 +428,10 @@ setup:
             term:
               attributes.host: "es-node-1"
   - match: { hits.total.value: 1 }
+
+  - do:
+      cluster.health:
+        wait_for_events: languid
 
   - do:
       indices.get_mapping:
@@ -482,6 +498,10 @@ setup:
             term:
               resource.service: "my-service"
   - match: { hits.total.value: 1 }
+
+  - do:
+      cluster.health:
+        wait_for_events: languid
 
   - do:
       indices.get_mapping:
@@ -748,6 +768,10 @@ setup:
             term:
               attributes.host: "es-node-1"
   - match: { hits.total.value: 0 }
+
+  - do:
+      cluster.health:
+        wait_for_events: languid
 
   - do:
       indices.get_mapping:

